src/main/resources/locales/en/interim-applications-chd-daily-cause-list.json

```diff
@@ -0,0 +1,20 @@
+{
+    "title": "Interim Applications List (Chancery Division) Daily Cause List",
+    "heading": "Interim Applications List (Chancery Division) Daily Cause List",
+    "venueLine1": "Business and Property Courts Rolls Building",
+    "venueLine2": "7 Rolls Buildings, Fetter Ln, City of London",
```

Comment:
I have discussed with Kim and Ogechi on the other Rolls Building list ticket. The venue should be like the mocked up style guide, and not the venue shown in the attached Excel file. Also the first line of the venue 'Rolls Building' needs to be bold